### PR TITLE
Home

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
code looks good, feel free to merge

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
```
Comment (AshutoshM29) [inline src/App.css:17]: You use use var(color) here

File: src/Pages/CategoryPage.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            catch(err){
+                console.log(err)
+            }
+
+        }
+        getCategories()
+    },[]);
+    const filteredData = state.categories.filter(x=>x.name.includes(state.category))
+    console.log("filtered data",filteredData)
+    console.log("woww")
```
Comment (AshutoshM29) [inline src/Pages/CategoryPage.js:29]: You can remove console, if it's not important

File: src/Pages/home.css [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+}
+.question-container h1{
+    font-size: 4rem;
+    font-weight: 600;
+    text-align: center;
+    padding: 2rem;
+    padding-left:4rem;
+    width:100%;
+    padding-right: 4rem;
+    border:1px solid grey;
```
Comment (AshutoshM29) [inline src/Pages/home.css:267]: Again you should use var here.

**Review by AshutoshM29 — APPROVED** [review]
I would suggest you to use different colours for your app
Rest looks good to merge!

**Comment by rajaranjanmr** [comment]
pushed changes in next commit

**Pull request merged**